R/mod_downloadHandler.R

```diff
@@ -51,7 +51,8 @@ mod_downloadHandler_include_ui <- function(id){
         NS(id, "report_includes"), label = NULL, inline = FALSE,
         choices = my_choices, selected = my_choices
       )
-    )
+    ),
+    actionButton(NS(id, "save_choices"), "Save Choices")
```

Comment:
I'm not saying I'm sold on this approach yet, but I like the "Store Preferences" language a little more than "Save Choices".
```suggestion
    actionButton(NS(id, "save_choices"), "Store Preferences")
```

Reply:
Updated this